# Only compile the molden binary to keep things simple
# By Toon Verstraelen (toon.verstraelen@ugent.be)
--- molden5.0-orig/makefile	2013-11-28 16:02:04.563545000 +0100
+++ molden5.0/makefile	2013-11-28 16:08:13.606238000 +0100
@@ -252,7 +252,7 @@
 #
 #LIBSG = -L/usr/X11R6/lib -Wl,-framework -Wl,GLUT -Wl,-framework -Wl,OpenGL -Wl,-framework -Wl,Cocoa -lGLU -lGL -lXmu -lX11 -lm
 
-all:	molden gmolden ambfor/ambfor ambfor/ambmd surf/surf
+all:	molden
 
 
 molden:	$(OBJS) mpdum.o xwin.o 
